src/message/MessageFull.js

```diff
@@ -49,3 +55,10 @@ export default class MessageFull extends React.PureComponent {
     );
   }
 }
+
+export default connect(
```

Comment:
Good catch.  I think we did use it before, but not anymore. Would be nice for an eslint rule to exist for this. Will remove.